src/wallet/wallet.cpp

```diff
@@ -53,7 +53,7 @@ const std::map<uint64_t,std::string> WALLET_FLAG_CAVEATS{
     },
 };
 
-static const size_t OUTPUT_GROUP_MAX_ENTRIES = 10;
+static constexpr size_t OUTPUT_GROUP_MAX_ENTRIES = 100;
```

Comment:
There is a [comment](https://github.com/fjahr/bitcoin/blob/241e14162fdfdddf697536113a68a3b11449db63/src/wallet/wallet.cpp#L2500) that became outdated and should maybe just refer to OUTPUT_GROUP_MAX_ENTRIES:
`// Cases where we have 11+ outputs all pointing to the same destination may result in privacy leaks as they will potentially be deterministically sorted.`

Reply:
Very nice find! Fixed!